kod/util/system.kod

```diff
@@ -4382,9 +4382,21 @@ messages:
 
    RecreateParliament()
    {
+      local plSavedFlagStates;
+
       if poParliament <> $
       {
+         plSavedFlagStates = Send(self,@ListCopy,#source=Send(Send(self,@GetTerritoryGame),@GetFlagStates));
```

Comment:
Why is copying the list necessary here?  Does Recreate modify the list in place?

Reply:
What I'm seeing (now that I'm looking more closely) is that Recreate overwrites flag states. It clears and creates new flags. Following the breadcrumbs, I see that `SetFaction` leads to `FlagClaimed` and then to `SaveFlagStates`, where the flag states are replaced.

I'm going to need at this with fresh eyes another time because I just went cross-eyed figuring this out.

Reply:
My understanding is that Recreate creates a new set of flags (`PlaceFlagsFullGame()`) and in the process replaces `plFlagStates` with the starting states, by way of `SetFaction()` to `FlagClaimed()` to `SaveFlagStates()`. I'm not sure what to do with this information yet.

Reply:
(I think) that process builds up a new list and does not modify the old one.  If so, you wouldn't need to make a copy of the old list here.  (Note that ListCopy is used only once or twice in the whole codebase, in very unusual circumstances where a list is chopped up into pieces.)